pages/new.js

```diff
@@ -0,0 +1,9 @@
+import Form from "../components/Form/Form";
+
+export default function NewPage() {
+  return (
```

Comment:
```suggestion
export default function NewPage(onNewCardCreation) {

you pass the new card data to the app and create a state where you bring the new card and the initial Data array together :)

I think this is crucial to complete this US, because otherwise the user doesn't know they created a new card (users don't know how to open the console.


return (
```